### DISCUSSION TITLE
Can Cytoscape open multiple views for a single data file?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [cytoscape/cytoscape.js#3296] [Q&A]
Can Cytoscape open multiple views for a single data file? 

For example, in view1 I view layout 1, and in view2 I view layout 2.

![20241111114733](https://github.com/user-attachments/assets/c67c8fde-3a58-411f-bee3-c923d9e88bac)

**Comment by mikekucera** [comment]
Please post questions about Cytoscape desktop to the cytoscape-helpdesk google group: https://groups.google.com/g/cytoscape-helpdesk
